components/automate-elasticsearch/habitat/hooks/post-run

```diff
@@ -0,0 +1,36 @@
+#!{{pkgPathFor "core/bash"}}/bin/bash
+
+exec 2>&1
+
+# ElasticSearch >= 6 doesn't allow configuring index settings in the elasticsearch.yml
+# so we need to configure these after ElasticSearch is up.
+
+# The post-run hook is fired immediately after run and doesn't wait until
+# elasticsearch is healthy. Therefore, we'll loop in the health-check until
+# it's green (0) or yellow (1) before we try and make the request.
+
+{{#if svc.me.cfg.http-host ~}}
+HOST="{{svc.me.sys.ip}}"
+{{else}}
+HOST="127.0.0.1"
+{{/if ~}}
+
+source {{pkg.svc_config_path}}/health_check
+
+wait_until_healthy 300 # seconds
```

Comment:
i think past experience has shown blocking in hooks is not good. https://github.com/habitat-sh/habitat/pull/6705 indicates that it will retry if you exit with non-zero